packages/astro/src/cli/index.ts

```diff
@@ -171,64 +118,56 @@ async function handleConfigError(
  **/
 async function runCommand(cmd: string, flags: yargs.Arguments) {
 	const root = flags.root;
-	// logLevel
-	let logging: LogOptions = {
-		dest: nodeLogDestination,
-		level: 'info',
-	};
+
+	// These commands can run directly without parsing the user config.
 	switch (cmd) {
 		case 'help':
 			printAstroHelp();
-			return process.exit(0);
+			return;
```

Comment:
I don't think we need the `process.exit` here to short-circuit the CLI. The continuing paths after a `return` should immediately exit node too. Exit code should still be 0 because of https://github.com/withastro/astro/blob/34918c9e043ac434d219910ea5599491f2504f22/packages/astro/astro.js#L74-L76